test/integration/admin_has_a_private_dashboard_test.rb

```diff
@@ -0,0 +1,75 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.